todo/src/components/TodoForm.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button, Form, FormGroup, Label, Input } from 'reactstrap';
+
+const TodoForm = props => {
+	return (
+		<div>
+			<Form>
+				<FormGroup>
+					<Label for="Add A Todo">Add a Todo </Label>
+					<div className="form">
+						<Input
+							name="todoText"
+							value={props.todoText}
+							type="text"
+							onChange={props.handleInputChange}
+							placeholder="Enter here"
+						/>
+						<Button color="primary" onClick={props.addTodo}>
```

Comment:
Ran into a bug when I press enter to submit a todo, the whole app refreshes. If I click the button things work as expected.